packages/storefront-lwc/src/modules/commerce/productSearchResults/productSearchResults.js

```diff
@@ -5,38 +5,60 @@
     For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
 */
 import { LightningElement, wire, track, api } from 'lwc';
-import { productsByQuery } from 'commerce/data';
+import QUERY from './gqlTemplate';
+import { useQuery } from '@lwce/apollo-client';
+import { apiClient } from '../api/client';
 
-//
-// Displays search results
-//
 export default class ProductSearchResults extends LightningElement {
     @track state = {};
     @track products = [];
     @track refinementgroups = [];
-    @api query = '';
     @track loading = false;
     @track refinementBar = 'refinement-bar col-md-3 d-none d-lg-block';
     @track showRefinementBar = true;
-    sortRule;
     selectedRefinements = {};
 
-    // The wire adaptor to search for products when the query, sort rule or selected refinements change.
-    @wire(productsByQuery, {
-        query: '$query',
-        sortRule: '$sortRule',
-        selectedRefinements: '$selectedRefinements',
+    variables = {
+        query: '',
+        filters: [],
+    };
+    sortRuleValue = '';
+
+    @api set query(val) {
+        this.variables = { ...this.variables, query: val };
+        console.log(this.variables);
+    }
+
+    get query() {
+        console.log(this.variables);
+        return this.variables.query;
+    }
+
+    @api set sortRule(val) {
+        this.sortRuleValue = val;
+        const filters = this.getFilters();
+        this.variables = { ...this.variables, filters: filters };
+    }
+
+    get sortRule() {
+        return this.sortRuleValue;
+    }
+
+    @wire(useQuery, {
+        query: QUERY,
+        lazy: false,
+        variables: '$variables',
     })
-    updateProducts(json) {
-        // The method to handle the json results returned from the above wire adaptor.
-        if (json.data && json.data.productSearch) {
-            this.products = json.data.productSearch.productHits || [];
+    updateProducts(response) {
+        // The method to handle the response results returned from the above wire adaptor.
```

Comment:
Please update this comment to not say wire adaptor since we are no longer using it.

Reply:
There is still a wire adaptor - useQuery is the wire adaptor